### ISSUE TITLE
android:allowBackup=true is back in Release 3.0.0

### ISSUE THREAD

**Issue by oorlov** [launchdarkly/android-client-sdk#138]
**Describe the bug**
Release 3.0.0 added `android:allowBackup=true` back to the AndroidManifest.xml
Because of this app developers have to use `tools:replace="android:allowBackup"` as a workaround.
In general libraries are not supposed to declare this specific flag (unless there is some special use for it? Maybe?)

**To reproduce**
- Create an Android app with Android Studio default layout
- Add `allowBackup=false` into `application` within `AndroidManifest.xml`
- Add `implementation 'com.launchdarkly:launchdarkly-android-client-sdk:3.1.0'` to the `app/build.gradle`
- Get a build error:
```
Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:29:9-36
	is also present at [com.launchdarkly:launchdarkly-android-client-sdk:3.1.0] AndroidManifest.xml:13:9-35 value=(true).
	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:21:5-103:19 to override.
```

**Expected behavior**
Adding `implementation 'com.launchdarkly:launchdarkly-android-client-sdk:3.1.0'` shouldn't cause build error due to `allowBackup` conflict

**Language version, developer tools**
AndroidStudio 4.1.3

**OS/platform**
OS X, but should be any

**Comment by oorlov** [comment]
Caused by this I believe:
https://github.com/launchdarkly/android-client-sdk/pull/126/files#diff-4ea1d5d45e25fbd99af946e3fed3b51b84b61585974e6056d92e25710d108376R8

**Comment by gwhelanLD** [comment]
Hi @Oorlov,

Thanks for the issue report. We'll release a patch to address this issue. For the moment, you can follow the instructions in the error message to include `tools:replace="android:allowBackup" as a workaround.

Thanks,
@gwhelanLD

**Comment by gwhelanLD** [comment]
Hi @oorlov,

This should be resolved in the [3.1.1 release](https://github.com/launchdarkly/android-client-sdk/releases/tag/3.1.1) of the SDK.

Thanks for the report!
@gwhelanLD

**Comment by oorlov** [comment]
@gwhelanLD Thank you! It works well!
